elasticai/creator/nn/quantized_grads/fixed_point/autograd.py

```diff
@@ -0,0 +1,95 @@
+from typing import Callable
+
+import torch
+from torch import Tensor
+
+from .quantize_to_fixed_point import quantize_to_fxp_hte, quantize_to_fxp_stochastic
+
+
+def factory_autograd_func(
```

Comment:
how about calling this `create_autograd_func` or `make_autograd_func`?

The term `factory_autograd_func` sounds misleading to me. I'd expect it to give me a factory for autograd functions.

Also i think the term factory is misused like crazy in most of python community, which leads to more and more people interpreting it differently. Heck, i've been guilty of this myself, mostly due to laziness.

This is what all but some python people will refer to as a factory

https://refactoring.guru/design-patterns/abstract-factory